acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerProcessRunner.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.dsl.signer.runner;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import tech.pegasys.eth2signer.dsl.signer.SignerConfiguration;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.ProcessBuilder.Redirect;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.collect.Lists;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.awaitility.Awaitility;
+
+public class Eth2SignerProcessRunner extends Eth2SignerRunner {
+
+  private static final Logger LOG = LogManager.getLogger();
+  private static final Logger PROCESS_LOG =
+      LogManager.getLogger("tech.pegasys.eth2signer.SubProcessLog");
+
+  private final ExecutorService outputProcessorExecutor = Executors.newCachedThreadPool();
+
+  private Process process;
+
+  public Eth2SignerProcessRunner(final SignerConfiguration signerConfig) {
+    super(signerConfig);
+  }
+
+  @Override
+  protected void startExecutor(final List<String> params) {
+    final StringJoiner javaOpts = new StringJoiner(" ");
+
+    final String[] paramsAsArray = params.toArray(new String[params.size()]);
```

Comment:
Intellij is recommending to use toArray without sizing i.e. new String[0]. Their rationale seems sensible enough so think we go with that.

Reply:
done